database/sql/V048__create_settlement_rewards.sql

```diff
@@ -0,0 +1,29 @@
+-- Populated before settlement occurs on chain
+-- (after the submissions have been ranked)
+CREATE TABLE settlement_scores (
+  settlement_id bytea NOT NULL,
+  winning_score numeric(78,0) NOT NULL,
+  reference_score numeric(78,0) NOT NULL
+);
+
+-- Populated after block finalization via transactionReceipt.
+CREATE TABLE settlement_observations (
+  -- the appended settlement ID read from the call data
+  settlement_id bytea NOT NULL,
+  -- the transaction hash and log index to be able to `JOIN` on the `settlement`
+  -- table, read from the transaction receipt
+  tx_hash bytea NOT NULL,
+  log_index bigint NOT NULL,
+  gas_used numeric(78,0) NOT NULL,
+  effective_gas_price numeric(78,0) NOT NULL,
```

Comment:
Which `gas_price` is used here?
1. while building the auction
2. while adding the observed settlement
3. price of the block the settlement occurred

Do we maybe need multiple?

Reply:
The only gas price that matters to me is the gas price that is used to compute the transaction cost. 


<img width="949" alt="Screenshot 2023-02-07 at 10 28 25" src="https://user-images.githubusercontent.com/11778116/217205350-4e24db67-2eb8-4fe1-b5d6-23b4c43805c6.png">

Reply:
> Which gas_price is used here?

The transaction's "effective gas price". This has a specific meaning post-EUP-1559 to be the effective/actual gas price that was changed to the EOA for the transaction.